launcher/ui/dialogs/ReviewMessageBox.cpp

```diff
@@ -0,0 +1,31 @@
+#include "ReviewMessageBox.h"
+#include "ui_ReviewMessageBox.h"
+
+ReviewMessageBox::ReviewMessageBox(QWidget* parent, QString const& title, QString const& icon)
+    : QDialog(parent), ui(new Ui::ReviewMessageBox)
+{
+    ui->setupUi(this);
+}
+
+ReviewMessageBox::~ReviewMessageBox()
+{
+    delete ui;
+}
+
+auto ReviewMessageBox::create(QWidget* parent, QString&& title, QString&& icon) -> ReviewMessageBox*
+{
+    return new ReviewMessageBox(parent, title, icon);
+}
+
+void ReviewMessageBox::appendMod(const QString& name, const QString& filename)
+{
+    auto itemTop = new QTreeWidgetItem(ui->modTreeWidget);
+    itemTop->setText(0, name);
+
+    auto filenameItem = new QTreeWidgetItem(itemTop);
+    filenameItem->setText(0, QString("Filename: %1").arg(filename));
```

Comment:
wait, does this need tr()? Or does QString already do this for us?

Reply:
nvm seems like QString does this already

Reply:
hmm not sure to be honest. Can't find any info about this

Reply:
I found this: https://doc.qt.io/qt-5/i18n-source-translation.html#using-numbered-arguments
I'll use it from now on, it's much less confusing xD